app/src/main/java/com/example/fitappa/Authentication/DatabaseConstants.java

```diff
@@ -0,0 +1,34 @@
+package com.example.fitappa.Authentication;
+
+/**
+ * Constants to be used for Firebase methods
+ */
+public class DatabaseConstants {
+
+    /**
+     * Get the collection name for users
+     *
+     * @return String representing collection name for users
+     */
+    public String getUsersCollection() {
+        return "users";
```

Comment:
arent constants supposed to be all capital?

Reply:
constant variable names are supposed to be capital not string values. unless you mean the method name should be capital?